src/index.html

```diff
@@ -21,7 +21,14 @@
       >
         Create Tasks
       </button>
-
+      <!-- Temporary Button -->
+      <button
+        class="create-task-btn"
+        type="button"
+        onclick="location.href='featureFlag/index.html'"
+      >
+        Feature Flag
+      </button>
```

Comment:
Why this is a temporary button ??
If this will be removed from where we will be able to access this ??

Reply:
We can use the link directly as done for profile diffs. I'll remove this as we are still using mock data.

Reply:
Couldn't get your point, why remove it, when you will be adding it later after API integration

Reply:
I want this PR to be merged on develop branch before doing API integration so that PR doesn't grow any further. Currently, it's functioning on mock data so showing the button for feature flag is not a good ideal.

Reply:
But if you think it's not necessary then I'll remove the comment.

Reply:
@pallabez Ok means you don't want this page to be accessed from the UI until the backend APIs are complete. Make sense, we can go with this.